app/styles/_app.scss

```diff
@@ -22,4 +30,4 @@
     padding-right: 8px;
     padding-bottom: 4px;
     padding-left: 8px;
-}
\ No newline at end of file
+}

```

Comment:
Once these changes are merged in, we will be able to remove this css class (govuk-tag).

Reply:
yeah I'm working on extracting the foundation tag at the moment

Reply:
Brilliant